File: 10_BubbleSort/10.py
def ordenarBurbuja(a):
    for i in range(len(a)):
        for j in range(len(a) - 1):
            if(a[j] > a[j + 1]):
                a[j], a[j + 1] = a[j + 1], a[j] # swap

print("Escribe los elementos separados por un espacio: ", end="")
a = list(map(int, input().split()))
ordenarBurbuja(a)
print(a[:])

File: 10_BubbleSort/10.rb
def bubbleSort(a)
    # Basically compares all with all
    0.upto(a.length - 1) do |i|
        0.upto(a.length - 2) do |j|
	    # Upward
            if a[j] > a[j+1]
		# Swap
                aux = a[j]
                a[j] = a[j + 1]
                a[j + 1] = aux
            end
        end
    end
end

puts "Escribe los elementos separados por un espacio: "
a = gets.scan(/\d+/).map(&:to_i)
bubbleSort(a)
print a
